tests/test_create_sentence.rs

```diff
@@ -134,10 +134,8 @@ fn test_post_sentence_with_used_uuid_returns_409() {
     let sentence_iso639_3 = "eng";
     connection.insert_language(&sentence_iso639_3);
 
-    let sentence_uuid = uuid::Uuid::new_v4();
     let sentence_text = "This is one sentence.";
-    connection.insert_sentence(
-        &sentence_uuid,
+    let sentence_uuid = connection.insert_sentence(
         &sentence_text,
```

Comment:
as we know we're talking about a sentence, why not just naming it 'text' / 'second_text' , so that things would easily fit on one line ?